src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -126,6 +133,16 @@ def strict_deps(self):
     return self.payload.strict_deps
 
   @property
+  def exports(self):
+    """A list of exported libraries, which will be accessible to dependents.
+
+    :return: See constructor.
+    :rtype: list
+    """
+    exports = [Address.parse(spec, relative_to=self.address.spec_path) for spec in self.payload.exports]
+    return [self._build_graph.get_target(addr) for addr in exports]
```

Comment:
IIRC, this will only work if these exports are definitely in the context. This should always be the case when the `exports` are a subset of a target's `dependencies`, but that isn't currently being checked.

Another way to ensure that exports and dependencies intersect might be to declare exports as `traversable_dependency_specs` of a target, which would cause them to be _added_ to the dependencies of the target after construction: basically, the complete set of dependencies would be `exports+dependencies`, and so you wouldn't need to repeat them.